.github/actions/run-tests/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
       shell: bash -el {0}
 
     - name: Publish test results
-      uses: actions/upload-artifact@v3
```

Comment:
Highlighting this change in case we want to move this out to a separate backport.